docs/enterprise.md

```diff
@@ -61,3 +61,192 @@ On top of this, enterprise package of VictoriaMetrics includes the following imp
+
+In order to run binary release of VictoriaMetrics enterprise, download the release for your OS and unpack it.
+Then run `victoria-metrics-enterprise` binary from the unpacked directory.
+
+Before vX.Y.Z all the enterprise apps required `-eula` command-line flag to be passed to them.
+This flag acknowledges that your usage fits one of the cases listed above.
+
+After vX.Y.Z either `-eula` flag or the following flags are used:
+```console
+  -license string
+        See https://victoriametrics.com/products/enterprise/ for trial license
+  -license-file string
+        See https://victoriametrics.com/products/enterprise/ for trial license
+  -license.forceOffline
+        Force offline verification of license code. License is verified online by default. This flag runs license verification offline.
+```
+
+For example, the following command runs `victoria-metrics-enterprise` binary with the specified license:
+```console
+wget https://github.com/VictoriaMetrics/VictoriaMetrics/releases/download/vX.Y.Z/victoria-metrics-linux-amd64-vX.Y.Z-enterprise.tar.gz
+tar -xzf victoria-metrics-linux-amd64-vX.Y.Z-enterprise.tar.gz
+./victoria-metrics-prod -license={VM_KEY_VALUE}
+```
+
+Alternatively, the license can be specified via `-license-file` command-line flag:
+```console
+./victoria-metrics-prod -license-file=/path/to/license/file
+```
+
+The license file must contain the license key.
+
+### Docker images
+
+Docker images for VictoriaMetrics enterprise are available [here](https://hub.docker.com/u/victoriametrics).
+Enterprise docker images have `enterprise` suffix in their names. For example, `victoriametrics/victoria-metrics:vX.Y.Z-enteprise`.
+
+In order to run docker image of VictoriaMetrics enterprise component it is required to provide the license key command-line
+flag similar to the one described in the previous section.
+
+For example, the following command runs [VictoriaMetrics single-node](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html) docker image with the specified license:
+```console
+docker run --name=victoria-metrics victoriametrics/victoria-metrics:vX.Y.Z -license={VM_KEY_VALUE}
+```
+
+Alternatively, the license can be specified via `-license-file` command-line flag:
+```console
+docker run --name=victoria-metrics -v /vm-license:/vm-license  victoriametrics/victoria-metrics:vX.Y.Z -license-file=/vm-license
+```
+
+### Helm charts
+
+Helm charts for VictoriaMetrics components are available [here](https://github.com/VictoriaMetrics/helm-charts).
+
+In order to run VictoriaMetrics enterprise helm chart it is required to provide the license key via `license` value in `values.yaml` file 
+and adjust the image tag to the enterprise one. 
+
+For example, the following values file for [VictoriaMetrics single-node chart](https://github.com/VictoriaMetrics/helm-charts/tree/master/charts/victoria-metrics-single)
+is used to provide key in plain-text:
+```yaml
+server:
+  image:
+    tag: vX.Y.Z-enterprise 
```

Comment:
let's provide the working example with real version

Reply:
Yep, the idea is once we will have the first version which supports new license keys all entries of `X.Y.Z` on this page will be replaced by a real version